config/schema/pathauto_pattern.schema.yml

```diff
@@ -0,0 +1,36 @@
+pathauto.pathauto_pattern.*:
+  type: config_entity
+  label: 'Pathauto pattern config'
+  mapping:
+    id:
+      type: string
+      label: 'ID'
+    label:
+      type: label
+      label: 'Label'
+    uuid:
+      type: string
+    type:
+      type: string
+      label: 'Pattern type'
+    pattern:
+      type: string
+      label: 'Pattern'
+    selection_criteria:
```

Comment:
A bit worried that all this logic will make alias generation considerably slower. Would be good to do some basic profiling, e.g. generate 100 aliases or so.